core/src/main/scala/kafka/common/Topic.scala

```diff
@@ -19,7 +19,7 @@ package kafka.common
 
 import util.matching.Regex
 import kafka.coordinator.GroupCoordinator
-
+import org.apache.kafka.common.errors
```

Comment:
I needed to do this because there is an InvalidTopicException in kafka.common that gets picked up unless I use errors.InvalidTopicException. If you like I can fully qualify the usage and get rid of the import all together.